todoman/cli.py

```diff
@@ -97,6 +102,35 @@ def command_wrap(*a, **kw):
     return command_wrap
 
 
+class AppContext:
+    def __init__(self):
+        self.config = None
+        self.db = None
+        self.porcelain = False
+
+    @cached_property
```

Comment:
Are there any advantages to lazy-initialization? Is there any command that **doesn't** use a formatter?